src/tests/ftest/soak/soak.py

```diff
@@ -872,16 +894,15 @@ def run_soak(self, test_param):
         self.all_failed_jobs = []
         # cleanup soak log directories before test on all nodes
         result = slurm_utils.srun(
-            NodeSet.fromlist(self.hostlist_clients), "rm -rf {}".format(
-                self.log_dir), self.srun_params)
+            NodeSet.fromlist(self.hostlist_clients),
+            ["rm", "-rf", self.log_dir], self.srun_params)
```

Comment:
Do we need a full path for the `rm` command?

Reply:
not needed for rm

Reply:
not needed.